x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/organization_stats.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { ContentSection } from '../shared/content_section';
+import { ORG_SOURCES_PATH, USERS_PATH } from '../../routes';
+
+import { IAppServerData } from './overview';
+
+import { StatisticCard } from './statistic_card';
+
+export const OrganizationStats: React.FC<IAppServerData> = ({
+  sourcesCount,
+  pendingInvitationsCount,
+  accountsCount,
+  personalSourcesCount,
+  isFederatedAuth,
+}) => (
+  <ContentSection
+    title={
+      <FormattedMessage
+        id="xpack.enterpriseSearch.workplaceSearch.organizationStats.title"
+        defaultMessage="Usage statistics"
+      />
+    }
+    headerSpacer="m"
+  >
+    <div
+      className={`euiFlexGrid euiFlexGrid--gutterLarge euiFlexGrid--${
+        isFederatedAuth ? 'halves' : 'fourths'
+      }`}
+    >
+      <StatisticCard title="Shared sources" count={sourcesCount} actionPath={ORG_SOURCES_PATH} />
```

Comment:
There is no I18n on these titles because `EuiCard` only accepts strings as titles